shell/browser/web_contents_preferences.cc

```diff
@@ -137,7 +137,7 @@ WebContentsPreferences::WebContentsPreferences(
                 "electron");
   }
   SetDefaultBoolIfUndefined(options::kContextIsolation, false);
-  SetDefaultBoolIfUndefined(options::kWorldSafeExecuteJavaScript, false);
+  SetDefaultBoolIfUndefined(options::kWorldSafeExecuteJavaScript, true);
```

Comment:
Will it work when this code is executed?
https://github.com/electron/electron/blob/e641a747fb8bc5a7115e4b0581b756aca039b864/shell/browser/api/electron_api_web_contents_view.cc#L112-L113

Reply:
@MarshallOfSound related to #18369

Reply:
you need to use `IsEnabled(options::kWorldSafeExecuteJavaScript, true);` as well